src/views/Prisoners.jsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import { Route } from 'react-router-dom';
+import PrisonersList from '../components/PrisonersList';
 
 export class Prisoners extends React.Component {
     render() {
         return (
             <div className="prisoners-container">
                 {
                     this.props.prisons.map(prison => {
-                        <Route path={`/prisons/${prison.name}`} render={/* to fill */} />
+                        <Route path={`/prisons/${prison.name}`} render={() => (
```

Comment:
Remember the attribute coming from the server will be "location" not "name" just to give you a heads up.